server/src/main/java/com/octopus/teamcity/opentelemetry/server/LogMasker.java

```diff
@@ -5,15 +5,15 @@
 
 public class LogMasker {
 
+    public static final String API_KEY_REPLACEMENT_REGEX = "XXXXXXXXXXXXXXXX";
+
     private LogMasker() {
         throw new IllegalStateException("Utility class LogMasker should not be instantiated ");
     }
 
     public static String mask(String message) {
         final String API_KEY_REGEX = "([a-z0-9]{31})";
```

Comment:
Consider naming this to match the API we are trying to mask
```suggestion
        final String HONEYCOMB_API_KEY_REGEX = "([a-z0-9]{31})";
```

Reply:
updated this one!